opentelemetry-sdk/tests/test_configurator.py

```diff
@@ -18,19 +18,19 @@
 from unittest.mock import patch
 
 from opentelemetry import trace
-from opentelemetry.environment_variables import (
-    OTEL_PYTHON_ID_GENERATOR,
-    OTEL_TRACES_EXPORTER,
-)
+from opentelemetry.environment_variables import OTEL_PYTHON_ID_GENERATOR
 from opentelemetry.sdk._configuration import (
     _EXPORTER_OTLP,
     _EXPORTER_OTLP_SPAN,
     _get_exporter_names,
     _get_id_generator,
+    _import_exporters,
     _import_id_generator,
     _init_tracing,
 )
+from opentelemetry.sdk._logs.export import ConsoleExporter
```

Comment:
Not in this PR; Should we rename this to `ConsoleLogExporter`?

Reply:
It's a good question... this would allow users to import a console exporter for each signal more easily than having to import as another name if they were consistent. I do find ConsoleExporter nicer though, as the logs is already part of the import path. Happy to open an issue for renaming this if you'd like.

Reply:
No strong preference but I think explicit about naming is better.

Reply:
Fixed with https://github.com/open-telemetry/opentelemetry-python/pull/2307